packages/app/src/store/main-store.ts

```diff
@@ -13,12 +14,19 @@ export const useMainStore = defineStore({
   id: 'main',
   state: (): MainStoreState => {
     return {
-      navBarExpanded: true,
+      _navBarExpanded: true,
```

Comment:
Why is this prefixed with `_`? Seems to serve no purpose, it's not really a private variable.

Reply:
It has to have a different name than the exposed variable and should never be updated on it's own. 
How else can I achieve this private state member in pinia?

Reply:
I renamed it so that the flag can be set in tests.